.github/workflows/ci.yml

```diff
@@ -52,6 +52,10 @@ jobs:
 
       - name: Install packages
         run: pdm install
+      - name: Check Style
```

Comment:
No need this, lint checks will be done by pre-commit ci

Reply:
Removed this step from ci, as pre-commit usually installed global by pipx and contributors should be able to make style before create PR.